pkg/image/fetcher_test.go

```diff
@@ -213,6 +216,20 @@ func testFetcher(t *testing.T, when spec.G, it spec.S) {
 						_, err := imageFetcher.Fetch(context.TODO(), repoName, image.FetchOptions{Daemon: true, PullPolicy: image.PullAlways, Platform: "some-unsupported-platform"})
 						h.AssertError(t, err, "unknown operating system or architecture")
 					})
+
+					when("remote platform does not match", func() {
+						it.Before(func() {
+							h.SkipIf(t, runtime.GOOS != "linux", "Skipped on non-linux")
```

Comment:
I didn't know how to avoid the error `image operating system windows cannot be used on this platform` when running tests on LCOW

Reply:
done